README.md

```diff
@@ -43,8 +43,121 @@ Terraform module for Kyma uses the following terraform [providers](provider.tf),
 | cluster_id          | cluster ID of the created Kyma environment                                                                                 |
 | domain              | domain of the created Kyma environment                                                                                     |
 
-## Running `terraform-sap-kyma-on-btp` module
+
+## How to use terraform module for kyma
+
+Please refer to the included usage [examples](./examples/).
+
+In order to use the module you need to create a dedicated folder in your project repository (for example `tf`). You will need to create a main terraform file (`main.tf`) and  `.tfvars` file. This will become the so called root terraform module where the module for kyma can be used as a child module.
```

Comment:
```suggestion
1. To use the module, create a dedicated folder in your project repository (for example, `tf`) and a main Terraform (`main.tf`) and `.tfvars` files. This will become the so-called root Terraform module, where the module for Kyma can be used as a child module.
```